src/Responsive/Responsive.test.jsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import '../utils/reactResponsive.mock';
+
+import {
+  breakpoints,
+  ExtraSmall,
+  Small,
+  Medium,
+  Large,
+  ExtraLarge,
+  LargerThanExtraSmall,
+} from './index';
+
+describe('<ExtraSmall />', () => {
+  it('should render children for extra small displays', () => {
+    global.innerWidth = breakpoints.extraSmall.maxWidth;
+
+    const wrapper = mount((
+      <ExtraSmall>
+        <p>Hello world</p>
+      </ExtraSmall>
+    ));
+    expect(wrapper.find('p').length).toEqual(1);
```

Comment:
You can use `expect(<...>).toHaveLength(<num>);` here as well. Just throwing it out as an option, since Jest has a number of matchers to use.